compiler/ilgen/JitBuilderRecorderTextFile.hpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#ifndef TR_JITBUILDERRECORDER_TEXTFILE_INCL
+#define TR_JITBUILDERRECORDER_TEXTFILE_INCL
+
+#include "ilgen/OMRJitBuilderRecorderTextFile.hpp"
+
+namespace TR
+{
+   class JitBuilderRecorderTextFile : public OMR::JitBuilderRecorderTextFile
+      {
+      public:
+      //    JitBuilderRecorderTextFile(const TR::MethodBuilderRecorder *mb, const char *fileName)
```

Comment:
and here...I'll stop pointing it out now, but please change all the places where you commented the "correct" thing out and just remove the `Recorder` name in the class.